packages/realm/typedoc/theme.tsx

```diff
@@ -0,0 +1,84 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+import { Application, DefaultTheme, DefaultThemeRenderContext, JSX, Options, PageEvent, Reflection } from "typedoc";
+
+/**
+ * A clone of the default theme, which prints a message when rendering each page.
+ */
+export class LoggingTheme extends DefaultTheme {
+  render(page: PageEvent<Reflection>): string {
+    return super.render(page);
+  }
+}
+
+/**
+ * The theme context is where all of the partials live for rendering a theme,
+ * in addition to some helper functions.
+ */
+export class FooterOverrideThemeContext extends DefaultThemeRenderContext {
+  constructor(theme: DefaultTheme, options: Options) {
+    super(theme, options);
+
+    const oldFooter = this.footer;
+
+    // Overridden methods must have `this` bound if they intend to use it.
+    // <JSX.Raw /> may be used to inject HTML directly.
+    this.footer = () => {
+      return (
+        <>
+          {oldFooter()}
+          <div class="container">
+            <JSX.Raw html={this.markdown("Custom footer text, with **markdown** support!")} />
```

Comment:
Is this still in progress? 🙂

Reply:
this was a fail 🙃. i should delete the file